assays:
    - n_trials: 5
      trial_duration: 5.0
      force_field:
          enabled: false
          matrix: [0.0, 0.0, 0.0, 0.0]